.github/workflows/build.yml

```diff
@@ -48,6 +51,14 @@ jobs:
         run: python tools/sortKeymap.py -t config/${{ env.os }}/reaper-kb.ini
       - name: setup
         run: pip install scons
+      - id: get-scons-msvc-cache
+        name: Get SCons MSVC Cache
+        if: ${{ env.CACHE_KEY != '' }}
```

Comment:
I think this should check the os rather than the presence of CACHE_KEY.We might want one day want CACHE_KEY for Mac, so this seems like a potential easy-to-miss footgun.

Reply:
Addressed that.